roles/pulp_common/README.md

```diff
@@ -134,7 +134,21 @@ RPM packages instead if this variable is set. Other distro packaging formats may
   (RPM) "packages".
   Defaults to "pip".
 
-If it is set to "packages", the following variables are used, or behave *differently* from above:
+If it is set to "packages", the installer is in packges mode, which has the following **limitations**:
+
+* Only CentOS/RHEL 7 and CentOS/RHEL 8 are supported.
```

Comment:
this wording contradicts with the "thus not officially supported by the Foreman project"

Should you say "the packages are built for …" instead?